tutorials/3d/merge_groups.rst

```diff
@@ -0,0 +1,248 @@
+the ``MergeGroup``. An example might be a steering wheel, or sail.
+
+Godot allows you fine control over exactly which nodes should be considered for
+merging, via the ``merging_mode`` property which is present in the inspector
+for every ``Spatial``, inside the ``Misc`` tab.
+
+Merging modes:
+
+-  ``Inherit`` - Inherit the setting from the parent. The default for the scene root node is ``On``.
+-  ``On`` - Change the mode of the node (and any inheriting children) to allow merging.
+-  ``Off`` - Change the mode of the node (and any inheriting children) to disallow merging.
+
+This means that if you, e.g. set the mode of a steering wheel to ``Off``, it
+will not be considered for merging, and neither will any children or
+grandchildren of the steering wheel (unless one of them explicitly reactivates
+merging with an ``On`` ``merge_mode``).
+
+
+Ways to use MergeGroups
+-----------------------
+
+There are three ways to use MergeGroups:
+
+AutoMerge
+^^^^^^^^^
+
+The ``MergeGroup`` will attempt to merge any descendent merges as it enters the
+``SceneTree``.
+This is the simplest method, and the best introduction to merging.
+
+Baking in the Editor
+^^^^^^^^^^^^^^^^^^^^
+
+When the ``MergeGroup`` is selected in the editor, a new ``bake`` button should
+appear in the toolbar. This allows you to bake the entire merged scene out to a
+file (.tscn (text scene) or .scn (binary scene)) at *design time* rather than at
+*runtime*. This can be used creatively to build sensible composite objects which
+are later used to compose a game level (this general approach is often known as
+"kitbashing").
+
+Additionally, baking in advance offers a great way to preview what will happen
+when merging at runtime. It allows you to open the merged scene, and see which
+meshes were successfully merged, and which were problematic.
+
+.. figure:: img/bake_merge_group.webp
+    :align: center
+
+Pros
+~~~~
+
+-  Allows you easily see the results of merging
+-  Allows you to further build levels using these merged scenes
+-  No time is taken by the merging process at runtime
+
+Cons
+~~~~
+
+-  Merged scenes typically require substantially more storage than the original meshes
+-  The size of your exported game will likely increase
+-  Larger merged scenes may take longer to load
+-  Larger merged scenes may take up more RAM at runtime, especially on the GPU
+
+If you merge, e.g. 10 boxes, the merged scene will have to store 10 times as
+much geometry data, as the polygons are duplicated. More storage means your
+exported game will be larger, the data will take longer to load, and consume
+more RAM at runtime. For this reason, baking in advance tends to be more
+practical with low poly meshes and art styles.
+
+.. tip:: Due to the increased storage requirements, it is recommended that, wherever possible, you bake scenes in binary format (.scn) rather than text (.tscn). This is because binary scenes are much more compact in terms of storage.
```

Comment:
```suggestion
.. tip::

    Due to the increased storage requirements, it is recommended that, wherever possible,
    you bake scenes in binary format (``.scn``) rather than text (``.tscn``).
    This is because binary scenes are much more compact in terms of storage,
    while also being faster to load and save.
```